src/kakuro/core/Tools.java

```diff
@@ -7,6 +7,9 @@
 import javax.swing.tree.TreeNode;
 
 /**
+ * @deprecated
+ * since iteration 2 as we are using preloaded games from the database
```

Comment:
it does not make the code of tools deprecated lol

Reply:
done